<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Telescope_control</class>
 <widget class="QWidget" name="Telescope_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telescope_control</string>
  </property>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>731</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>517</width>
     <height>539</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Main">
    <attribute name="title">
     <string>Automatic</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="server_connect_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Connect To WSP</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="chiller_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Turn On Chiller</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="startup_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Startup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="shutdown_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Shutdown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="restart_selection">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <item>
            <property name="text">
             <string>Total Restart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mount Restart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dome Restart</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="restart_button">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Restart</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLineEdit" name="RA_input">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>DEC (J2000)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="object_entry">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Object Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="observe_object_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Observe Object</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="exposure_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>DO EXPOSURE</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="exposure_input">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>RA (J2000)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QComboBox" name="filter_selection">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <item>
          <property name="text">
           <string>U Band Filter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R Band Filter</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="do_focus_loop">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Focus Loop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="plot_option">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>With Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QSlider" name="mount_tracking_toggle">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QSlider::groove:vertical {
    background: black;
    position: absolute; 
    left: 4px; right: 4px;
}

QSlider::handle:vertical {
    height: 10px;
    background: red;
    margin: 0 -4px; 
}

QSlider::add-page:vertical {
    background: black;
}

QSlider::sub-page:vertical {
    background: black;
}</string>
               </property>
               <property name="maximum">
                <number>1</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Mount Tracking</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="dome_tracking_toggle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QSlider::groove:vertical {
    background: black;
    position: absolute; 
    left: 4px; right: 4px;
}

QSlider::handle:vertical {
    height: 10px;
    background: red;
    margin: 0 -4px; 
}

QSlider::add-page:vertical {
    background: black;
}

QSlider::sub-page:vertical {
    background: black;
}</string>
             </property>
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Dome Tracking</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="DEC_input">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Exposure Time (seconds)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="goto_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Go To Coordinate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="10">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Status Bar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Chiller On:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QToolButton" name="chiller_status_light">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="7" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>CCD Temperature:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLCDNumber" name="ccd_temp_display">
           <property name="font">
            <font>
             <family>MathJax_Vector-Bold</family>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background:red;
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="6">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Dome Open:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QToolButton" name="dome_status_light">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>CCD On:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QToolButton" name="ccd_status_light">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="output_display">
       <property name="maximumSize">
        <size>
         <width>791</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 255, 255)</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Manual">
    <attribute name="title">
     <string>Manual</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QPushButton" name="update_button">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Enter a Command:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="command_entry"/>
         </item>
         <item>
          <widget class="QPushButton" name="command_execute">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: green
</string>
           </property>
           <property name="text">
            <string>Execute</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <widget class="QPlainTextEdit" name="output_display_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Current Housekeeping Statuses</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="output_display_2">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Last_Image">
    <attribute name="title">
     <string>Last Image</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
